godot-core/src/engine/script_instance.rs

```diff
@@ -575,27 +575,22 @@ mod script_instance_info {
             )
         };
 
-        let length = handle_panic(ctx, || {
+        let vec = handle_panic(ctx, || {
```

Comment:
Can this still panic? If yes, maybe consider commenting how.

Reply:
No, this should no longer panic.